### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by allenpg**
Copied/pasted the README.md template to the top of file; added AGPL v3; Dawn advised to flag ##Optional, ##Development, and ##Deployment to B%

### CODE REVIEW

**Comment by titaniumbones** [comment]
#8 makes changes to `readme.md` so I'm going to *WAIT* on this in order to avoid conflicts.

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+[![GitHub](https://img.shields.io/badge/project-Data_Together-487b57.svg?style=flat-square)](http://github.com/datatogether)
+[![Slack](https://img.shields.io/badge/slack-Archivers-b44e88.svg?style=flat-square)](https://archivers-slack.herokuapp.com/)
+[![License](https://img.shields.io/github/license/mashape/apistatus.svg)](./LICENSE) 
+
 --
     import "github.com/archivers-space/warc"
 
+## License & Copyright
+
+[Affero General Public License v3](http://www.gnu.org/licenses/agpl.html) ]
```
Comment (mhucka) [inline readme.md:14]: It looks like there's a stray `]` at the end of that line?

File: readme.md [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+
+## Getting Involved
+
+We would love involvement from more people! If you notice any errors or would like to submit changes, please see our [Contributing Guidelines](./.github/CONTRIBUTING.md). 
+
+We use GitHub issues for [tracking bugs and feature requests](https://github.com/datatogether/REPONAME/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/REPONAME/pulls)
```
Comment (mhucka) [inline readme.md:20]: Missing period at end of the sentence.

**Review by mhucka — COMMENTED** [review]
Super trivial copy editing comments :-)

**Review by b5 — APPROVED** [review]
I approve of this message. LGTM

**Comment by b5** [comment]
I'll rebase the other branch to get readme updates

**Pull request merged**